docs/source/databases.rst

```diff
@@ -189,3 +191,81 @@ writing, and would always bind to a master database with read/write
 privileges).  It's also possible to extend
 :class:`~pecan.hooks.TransactionHook` or write your own hook implementation for
 more refined control over where and when database bindings are called.
+
+Assuming a master/standby setup, where the master accepts write requests and
+the standby can only get read requests, a Pecan configuration for sqlalchemy
+could be::
+
+    # Server Specific Configurations
+    server = {
+        ...
+    }
+
+    # Pecan Application Configurations
+    app = {
+        ...
+    }
+
+    # Master database
+    sqlalchemy_w = {
+        'url': 'postgresql+psycopg2://root:@master_host/dbname',
+        'pool_recycle':  3600,
```

Comment:
This is kind of nitpick, but does the whitespace here (after the colon) get rendered in sphinx's output?